### ISSUE TITLE
Cannot Access Transactions List 

### ISSUE THREAD

**Issue by bettyvschmartz** [bisq-network/bisq#6379]
### Description

Transactions list view under funds > transactions is no longer accessible. Get an unhandled exception error;

```
Error
Cannot invoke "bisq.core.trade.model.Tradable.getId()" because the return value of "bisq.desktop.main.funds.transactions.TransactionsListItem.getTradable()" is null
```

No sure why this throws a NPE as Tradable seems like it can be nullable. 

#### Version

Version 1.9.5

### Steps to reproduce

Would assume this is possibly isolated, but can't be certain. 

### Expected behaviour

Transaction tab opens and all transactions are viewable. 

### Actual behaviour

The above null pointer exception. 

### Screenshots

N/A

#### Device or machine

Ubuntu 20.04

#### Additional info

Extra debug;

```
java.lang.NullPointerException: Cannot invoke "bisq.core.trade.model.Tradable.getId()" because the return value of "bisq.desktop.main.funds.transactions.TransactionsListItem.getTradable()" is null
	at bisq.desktop.main.funds.transactions.TransactionsView$2$1.updateItem(TransactionsView.java:399)
	at bisq.desktop.main.funds.transactions.TransactionsView$2$1.updateItem(TransactionsView.java:382)
	at javafx.scene.control.TableCell.updateItem(TableCell.java:665)
	at javafx.scene.control.TableCell.indexChanged(TableCell.java:469)
	at javafx.scene.control.IndexedCell.updateIndex(IndexedCell.java:120)
	at javafx.scene.control.skin.TableRowSkinBase.updateCells(TableRowSkinBase.java:539)
	at javafx.scene.control.skin.TableRowSkinBase.<init>(TableRowSkinBase.java:159)
	at javafx.scene.control.skin.TableRowSkin.<init>(TableRowSkin.java:89)
	at javafx.scene.control.TableRow.createDefaultSkin(TableRow.java:213)
	at javafx.scene.control.Control.doProcessCSS(Control.java:897)
	at javafx.scene.control.Control$1.doProcessCSS(Control.java:89)
	at com.sun.javafx.scene.control.ControlHelper.processCSSImpl(ControlHelper.java:67)
	at com.sun.javafx.scene.NodeHelper.processCSS(NodeHelper.java:145)
	at javafx.scene.Node.processCSS(Node.java:9547)
	at javafx.scene.Node.applyCss(Node.java:9634)
	at javafx.scene.control.skin.VirtualFlow.setCellIndex(VirtualFlow.java:1743)
	at javafx.scene.control.skin.VirtualFlow.addTrailingCells(VirtualFlow.java:2118)
	at javafx.scene.control.skin.VirtualFlow.layoutChildren(VirtualFlow.java:1247)
	at javafx.scene.Parent.layout(Parent.java:1207)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Parent.layout(Parent.java:1214)
	at javafx.scene.Scene.doLayoutPass(Scene.java:576)
	at javafx.scene.Scene$ScenePulseListener.pulse(Scene.java:2476)
	at com.sun.javafx.tk.Toolkit.lambda$runPulse$2(Toolkit.java:413)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at com.sun.javafx.tk.Toolkit.runPulse(Toolkit.java:412)
	at com.sun.javafx.tk.Toolkit.firePulse(Toolkit.java:439)
	at com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:563)
	at com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:543)
	at com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(QuantumToolkit.java:536)
	at com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$11(QuantumToolkit.java:342)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
	at java.base/java.lang.Thread.run(Unknown Source)
```

I could try restore an earlier backup but would rather not lose data. 

-- bvs

**Comment by ghost** [comment]
@bettyvschmartz thanks for the report, fixed in #6381

**Comment by bettyvschmartz** [comment]
Great @jmacxx, looks good. 

-- bvs
